lib/sqlite/sqlite3.h

```diff
@@ -146,9 +146,9 @@ extern "C" {
 ** [sqlite3_libversion_number()], [sqlite3_sourceid()],
 ** [sqlite_version()] and [sqlite_source_id()].
 */
-#define SQLITE_VERSION        "3.41.2"
-#define SQLITE_VERSION_NUMBER 3041002
-#define SQLITE_SOURCE_ID      "2023-03-22 11:56:21 0d1fc92f94cb6b76bffe3ec34d69cffde2924203304e8ffc4155597af0c191da"
+#define SQLITE_VERSION        "3.45.1"
```

Comment:
Can we make a separate PR with deps update and no other logic changes?

Reply:
There is, as mentioned this is rebased on top off https://github.com/stellar/stellar-core/pull/4226. I think your comments refer to changes in that PR.

Reply:
Sorry, missed the description somehow.